.travis/script.sh

```diff
@@ -114,6 +114,13 @@ build_cc1352() {
 }
 
 build_cc2538() {
+    git checkout -- . || die
+    git clean -xfd || die
+    mkdir build && cd build || die
+    cmake -GNinja -DCMAKE_TOOLCHAIN_FILE=examples/platforms/cc2538/arm-none-eabi.cmake -DOT_PLATFORM=cc2538 || die
```

Comment:
I believe you need to add a ``../openthread`` here (and to the posix one) to point to the source directory. If it works as-is then I'm wrong!

Reply:
Good catch. Updated.